test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
Ok great, last one (hopefully, sorry for bothering you with all these examples 😅)

Assume:

* `BlogSerializer` has `default_include 'posts.author'`
* `PostSerializer` has `default_include 'author'`
* Blog `has_many :posts`, Post `has_one :author` and `belongs_to :category`

What happens to cases 2 and 3? I'm particularly interested in the "conflict" between

    render json: Blog.first, include: 'posts.category'

and 

    # blog_serializer.rb
    default_include 'posts.author'